feature/gnmi/ate_tests/telemetry_basic_check_test/telemetry_basic_check_test.go

```diff
@@ -437,7 +444,7 @@ func TestComponentParent(t *testing.T) {
 func TestSoftwareVersion(t *testing.T) {
 	dut := ondatra.DUT(t, "dut")
 
-	cards := components.FindComponentsByType(t, dut, cpuType)
+	cards := components.FindComponentsByType(t, dut, supervisorType)
```

Comment:
@mojiiba I think this changes looks good to me. Any objection? Thanks. 

cc: @vosipchu